alerts-server/scalafmt.conf

```diff
@@ -0,0 +1,9 @@
+project.git = true
```

Comment:
This file is supposed to be named `.scalafmt.conf` (see the dot).

Reply:
Fixed. Thanks!